### DISCUSSION TITLE
Angular Hot Module Reload

### DISCUSSION THREAD

**Discussion by rubenmarcus** [frontendbr/forum#1119] [Perguntas e Respostas]
Alguém já conseguiu implementar no Angular essa feature do React? se sim ,pode dizer como?

Vi alguns plugins e tutoriais, e não deu muito certo aqui,

se tiverem dicas também de performance pra deixar a aplicação angular,mais rápida ( estou usando lazy loading nos modulos ) , seria bem vindo

**Comment by thismarcoantonio** [comment]
Se for relativo ao Angular mesmo (>=v1), segui este [repositorio](https://github.com/gdi2290/angular-hmr) uma vez, aplicando assim o hot module replacement.

Por outro lado, se for no angularjs, realmente nunca testei, mas temos este [package](https://github.com/noppa/ng-hot-reload) que deve funcionar;

**Comment by rubenmarcus** [comment]
@thismarcoantonio  funcionou no angular 5? seu processo de desenvolvimento ficou mais rapido? eu me sinto irritado, toda vez que eu salvo algo no projeto ter que recompilar e ler ele inteiro

**Comment by thismarcoantonio** [comment]
@rubenmarcus sim, testei foi no angular 5. Cara, ficou mais rápido sim, até pq eu usava redux e as vezes limpava a store
